net-im/spectrum2/spectrum2-2.0.7.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils user
+
+DESCRIPTION="An open source instant messaging transport"
+HOMEPAGE="https://www.spectrum.im"
+SRC_URI="https://github.com/SpectrumIM/spectrum2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc frotz irc mysql postgres purple sms +sqlite test twitter xmpp"
+REQUIRED_USE="|| ( mysql postgres sqlite )"
+
+DEPEND="dev-libs/boost:=
+	dev-libs/expat
+	dev-libs/log4cxx
+	dev-libs/jsoncpp:=
+	dev-libs/openssl:0=
+	dev-libs/popt
+	dev-libs/protobuf:=
+	net-dns/libidn
+	net-misc/curl
+	sys-libs/zlib:=
+	doc? ( app-doc/doxygen )
+	frotz? ( !games-engines/frotz )
+	irc? ( net-im/libcommuni )
+	mysql? ( || ( dev-db/mariadb-connector-c dev-db/mysql-connector-c ) )
+	postgres? ( dev-db/postgresql:= )
+	purple? ( dev-libs/glib
+		dev-libs/libev:=
+		net-im/pidgin:= )
+	sms? ( app-mobilephone/smstools )
+	sqlite? ( dev-db/sqlite:3 )
+	test? ( dev-util/cppunit )
+	twitter? ( net-misc/curl )
+	xmpp? ( net-im/swiften[ft]:= )"
```

Comment:
This is flagged up by the automated checks as an invalid dependency.

Reply:
Yes, I am aware of this. I changed it now to `net-im/swift`. But this package is also currently NOT in portage. Please see https://github.com/gentoo/gentoo/pull/9099. After talking to @Dessa, I switched  from `swiften` to `swift`, but there are a couple of problems open, which needs to be resolved before this can be commited..